yb-voyager/src/srcdb/source.go

```diff
@@ -289,3 +294,53 @@ func (source *Source) extrapolateDSNfromSSLParams(DSN string) string {
 
 	return DSN
 }
+
+func (source *Source) PrepareSSLParamsForDebezium(exportDir string) error {
+	switch source.DBType {
+	case "postgresql":
+		if source.SSLKey != "" {
+			targetSslKeyPath := filepath.Join(exportDir, "metainfo", "ssl", "key.der")
+			err := dbzm.WritePKCS8PrivateKeyPEMasDER(source.SSLKey, targetSslKeyPath)
+			if err != nil {
+				return fmt.Errorf("could not write private key PEM as DER: %w", err)
+			}
+			utils.PrintAndLog("Converted SSL key from PEM to DER format. File saved at %s", targetSslKeyPath)
+			source.SSLKey = targetSslKeyPath
+		}
+	case "mysql":
+		switch source.SSLMode {
+		case "disable":
+			source.SSLMode = "disabled"
+		case "prefer":
+			source.SSLMode = "preferred"
+		case "require":
+			source.SSLMode = "required"
+		case "verify-ca":
+			source.SSLMode = "verify_ca"
+		case "verify-full":
+			source.SSLMode = "verify_identity"
+		}
+		if source.SSLKey != "" {
+			keyStorePath := filepath.Join(exportDir, "metainfo", "ssl", "keystore.jks")
```

Comment:
is this the standard way of creating keystore and truststore files?
Just thinking shall we keep them hidden by starting their name with a `.`

Reply:
I think it's fine. Why hidden? 
For example, this is how an oracle wallet looks like:

```
~/oracle/wallet » ls                                                           
README       ewallet.p12  keystore.jks      sqlnet.ora    truststore.jks
cwallet.sso  ewallet.pem  ojdbc.properties  tnsnames.ora
```

Reply:
I was just thinking since we are creating a copy of some ssl params in a file, will it be safer to do that

Reply:
Makes sense. Also, printing to console, to make the user aware of this file generation.